docker/test/performance-comparison/compare.sh

```diff
@@ -1028,14 +1028,17 @@ do
         query_file=$(echo "$query" | cut -c-120 | sed 's/[/	]/_/g')
         echo "$query_file" >> report/query-files.txt
 
+        # All metainformation will be saved in flamegraph title and subtitle directly
+        flamegraph_filename=$(clickhouse-local -q "SELECT generateUUIDv4();")
```

Comment:
I still want to be able to open them locally, and this makes it impossible. Is the query text problematic? Let's use test name + query index instead. By the way, what about `query_file` variable, maybe you should just switch it to be test + query index? You can do this using `cut -f1-3`. Also not sure what is `report/query-files.txt`, maybe we should delete it.

Reply:
How I understand, the problem is in that current name has spaces and we have to encode it. Or use another name. `report/query-files.txt` is used when we generate differential flame graphs.

Reply:
Do spaces become a problem when you upload the files to S3? Removing query text should fix this.